controllers/ie_controller_test.go

```diff
@@ -0,0 +1,177 @@
+package controllers
```

Comment:
Sorry, I didn't explain myself well. I do agree that keeping the actual _file_ in the same _folder_ is a common idiom, the other is to make the actual test file _not_ be in the same _package_. There is a special rule in go to ignore the policy that every file in a folder be in a package with the same name as the folder. See point 2 of this article by Ben Johnson: https://medium.com/@benbjohnson/structuring-tests-in-go-46ddee7a25c#.3rhzlrh5i